datastore/state.go

```diff
@@ -0,0 +1,11 @@
+package datastore
+
+import "github.com/spacemeshos/go-spacemesh/sql"
+
+type LocalDB struct {
+	*sql.Database
+}
```

Comment:
lets store this code in some place other than datastore. maybe localsql or sql/local

Reply:
I moved the code to the `localsql` package and made it a bit easier to use.